charts/bazarr/0.4.1/questions.yaml

```
@@ -0,0 +1,375 @@
+groups:
+  - name: "Container Image"
+    description: "configure container image"
+  - name: "Configuration"
+    description: "additional container configuration"
+  - name: "Networking"
+    description: "Networking / service configuration"
+  - name: "Storage"
+    description: "configure app volume mounts"
+  - name: "Ingress"
+    description: "Ingress configuration"
+portals:
+  web_portal:
+    protocols:
+      - "http"
+    host:
+      - "$node_ip"
+    ports:
+      - "$variable-bazarrService.port"
```

Should I suggest this change when I see it

I don't see a `bazarrService.port` defined but I remember you said there was on going work around the portal being set based on ingress or not

```suggestion
      - "$variable-service.port.nodePort"
```